psa-crypto/src/operations/key_management.rs

```diff
@@ -8,8 +8,44 @@ use crate::types::key::{Attributes, Id};
 use crate::types::status::{Result, Status};
 use core::convert::TryFrom;
 
-/// Generate a key
-pub fn generate_key(attributes: Attributes, id: Option<u32>) -> Result<Id> {
```

Comment:
> How come you dropped the _key part of the name, because it's in key_management? 

Yes, that's the reason!

> Then you probably need to modify the examples to call it as key_management::generate(...), because it's confusing as to what generate is and where it is coming from.

Sure, will do.

Reply:
You've only modified the examples in this file, the other still import the function directly 🤡 but nevermind, we can discuss under #17